idea_town/frontend/static-src/app/views/scroll-header-view.js

```diff
@@ -0,0 +1,58 @@
+import app from 'ampersand-app';
+
+import BaseView from './base-view';
+import template from '../templates/scroll-header-view';
+const changeHeaderOn = 120;
```

Comment:
The airbnb style guide uses lowerCamelCase for consts.

I think the meaning of the const is clear from how it's used, FWIW.